conbench/templates/app.html

```diff
@@ -138,6 +138,32 @@ <h4 class="modal-title"><span class="glyphicon glyphicon-trash submenu"></span>&
     });
 }
 
+/*
+INFO:
+    Adds free text search ability for specific columns, based on selected column indexes.
+    Based on the indexes of the columns, input boxes are created. The search is triggered on
+    key press.
+INPUTS:
+ data_table_id: string
+ column_indexes: array
+ */
+function free_text_column_search(data_table_id, column_indexes) {
+    $(document).ready(function() {
+        var table = $(data_table_id).DataTable();
+        table.columns(column_indexes).every(function() {
+            var that = this;
+            var title = this.header().textContent;
+            $('<input class="custom-input-text custom-search-box" type="text" placeholder="🔍 ' + title + '" />')
```

Comment:
The use of emoji, facilitated to cap the width of the column + visual treat for `searchability` feature

Reply:
Love this!